spec/models/billing_entity_spec.rb

```diff
@@ -9,13 +9,19 @@
 
   it { is_expected.to belong_to(:organization) }
 
+  it { is_expected.to have_many(:customers) }
+  it { is_expected.to have_many(:invoices) }
+  it { is_expected.to have_many(:invoice_custom_section_selections) }
+  it { is_expected.to have_many(:selected_invoice_custom_sections) }
+  it { is_expected.to have_many(:fees) }
+
   describe "is_default validation" do
     let(:organization) { create :organization }
 
     it "validates uniqueness of organization_id for is_default excluding deleted and archived records" do
-      deleted_record = create(:billing_entity, :default, :deleted, organization:)
+      # by default an organization is built with a default billing entity
+      expect(organization.default_billing_entity.discard!).to be true
```

Comment:
what if the organization does not have a default billing entity? This is feasible as the billing entity could be discarded or archived...

Reply:
it's exactly the case, that in most of the cases, except when it's exactly a test case for billing_entity, we will and should have a default billing entity...
Shall we discuss it offline?

Reply:
discussed it offline, left a comment that it's needed temporary and will need to be deleted after the final implementation complete